numpy/random/mtrand.pyx

```diff
@@ -1154,7 +1154,7 @@ cdef class RandomState:
         probability density function:
 
         >>> import matplotlib.pyplot as plt
-        >>> count, bins, ignored = plt.hist(s, 15, density=True)
```

Comment:
I have a preference for leaving `mtrand.pyx` alone, even for documentation improvements like this.

Reply:
@rkern could you please explain you reasoning?

Reply:
[Per NEP 19](https://numpy.org/neps/nep-0019-rng-policy.html#supporting-unit-tests):

> It is intended that this class will no longer be modified, except to keep it working when numpy internals change.

Docstring improvements are borderline; nothing bad will happen if the docstrings are changed. Nonetheless, I'd rather keep a bright line and let the legacy `mtrand.pyx` file remain untouched except if triggered by some other change to numpy internals that would otherwise break it (e.g. we deprecate a numpy C API function that it uses).